tutorials/snap/setup-ubuntu-core-joule.md

```diff
@@ -0,0 +1,227 @@
+---
+id: setup-ubuntu-core-intel-joule
+summary: In this codelab, we are going to setup Ubuntu Core on your Intel Joule. We’ll flash the sd card with an Ubuntu Core image and then configure it.
+categories: snap
+status: Published
+feedback-link https://github.com/ubuntu/codelabs/issues
+tags: snap, usage, beginner, joule, console-conf
+difficulty: 1
+published: 2016-12-10
+Author: Didier Roche <didier.roche@canonical.com>
+
+---
+
+# Setup Intel Joule
+
+## Introduction
+Duration: 15:00
+
+Welcome to the world of Ubuntu Core! In this codelab, we are going to introduce you how to flash your eMMC for your Intel Joule board, and finally turn your Intel Joule into a Ubuntu Core 16 system.
+
+![IMAGE](https://assets.ubuntu.com/v1/3647779b-joule-product.png)
+
+
+### What you’ll learn
+  - Create a Ubuntu SSO (Single-Sign-On) account
+  - How to setup your hardware environment
+  - How to flash Ubuntu Core 16 for your Intel Joule board
+  - How to do console-conf for your Ubuntu Core 16 system
+### What you’ll need
```

Comment:
This list is rendered with ticks on your version, and as bullets in the original. I don't know if that's important, but I guess there's a newline missing above ### What you'll need